### PR TITLE
[Bugfix] fixed directory path of crediantials file in .post.py

### PR DESCRIPTION
**Pull request by tanmaydatta**

### CODE REVIEW

File: .credentials.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "username2":"password2"
     },
     {
-	"web-browser": "1"
+		"web-browser": "1"
```
Comment (yashmehrotra) [inline .credentials.json]: why the 4+8 indentation ?

File: .post.py [hunk 2]
```diff
@@ -17,7 +18,8 @@
 BROWSERS_DICT = {"0": "", "1": "firefox", "2": "google-chrome"}
 
 try:
-    credentials = load(open('.credentials.json'))
+    jsonPath = os.path.dirname(os.path.realpath(__file__))
+    credentials = load(open(jsonPath + '/.credentials.json'))
```
Comment (yashmehrotra) [inline .post.py]: Since we are already importing os, I think if you do ``from os import path`` that would be cool.

**Comment by yashmehrotra** [comment]
Thank you for the pull request. Please make the changes as discussed in the comments and we can the merge it :).

File: .post.py (hunk 2 above)
Comment (tanmaydatta) [inline .post.py]: from os import path *
doesn't work
os needs to be imported separately

File: .credentials.json (hunk 1 above)
Comment (tanmaydatta) [inline .credentials.json]: It was a mistake.
Will correct it.

File: .post.py (hunk 2 above)
Comment (yashmehrotra) [inline .post.py]: It is working on my terminal, are you sure it doesn't work ?

Comment (tanmaydatta) [inline .post.py]: from os import *
print os.path.dirname(os.path.realpath(__file__))

the above code is giving the following error:
Traceback (most recent call last):
  File "test.py", line 2, in <module>
    print os.path.dirname(os.path.realpath(__file__))
NameError: name 'os' is not defined

On Mon, Apr 13, 2015 at 11:36 AM, Yash Mehrotra <notifications@github.com>
wrote:

> In .post.py
> <https://github.com/526avijitgupta/cyberoam-rescue/pull/2#discussion_r28215993>
> :
>
> > @@ -17,7 +18,8 @@
> >  BROWSERS_DICT = {"0": "", "1": "firefox", "2": "google-chrome"}
> >
> >  try:
> > -    credentials = load(open('.credentials.json'))
> > +    jsonPath = os.path.dirname(os.path.realpath(__file__))
> > +    credentials = load(open(jsonPath + '/.credentials.json'))
>
> It is working on my terminal, are you sure it doesn't work ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/526avijitgupta/cyberoam-rescue/pull/2/files#r28215993>
> .
>



-- 
TANMAY DATTA

Comment (yashmehrotra) [inline .post.py]: Just use 
``from os import system, devnull, path``  
and use ``path.dirname`` instead of ``os.path.dirname``

Comment (yashmehrotra) [inline .post.py]: When you import * you are importing all the functions and classes present in the library, so you do not need to use the library name as a reference.

Comment (tanmaydatta) [inline .post.py]: Will correct all this.

On Mon, Apr 13, 2015 at 11:40 AM, Yash Mehrotra <notifications@github.com>
wrote:

> In .post.py
> <https://github.com/526avijitgupta/cyberoam-rescue/pull/2#discussion_r28216131>
> :
>
> > @@ -17,7 +18,8 @@
> >  BROWSERS_DICT = {"0": "", "1": "firefox", "2": "google-chrome"}
> >
> >  try:
> > -    credentials = load(open('.credentials.json'))
> > +    jsonPath = os.path.dirname(os.path.realpath(__file__))
> > +    credentials = load(open(jsonPath + '/.credentials.json'))
>
> When you import * you are importing all the functions and classes present
> in the library, so you do not need to use the library name as a reference.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/526avijitgupta/cyberoam-rescue/pull/2/files#r28216131>
> .
>



-- 
TANMAY DATTA

File: .credentials.json (hunk 1 above)
Comment (yashmehrotra) [inline .credentials.json]: Would you mind making a commit to fix the indentation, we prefer to have a clean code base to make it easy for other contributors to start.

**Comment by yashmehrotra** [comment]
Looks good, gonna merge it. BTW, next time it would be better if you could squash all these changes into one commit rather than 4.

**Pull request merged**